examples/slice-pizza/datadog-app/src/components/PizzasList.tsx

```diff
@@ -46,7 +50,9 @@ export function PizzaLists(props: { onSubmitOrder: any; token: Token }) {
         setPizzas(updatedPizzas);
     };
 
-    const onSubmit = () => {
+    const onSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+        e.preventDefault();
```

Comment:
I spent way more time than I'd like to admit debugging an issue where the `/cart` `POST` requests were cancelled after placing the order because I had converted the page to a form and wasn't preventing the submit from redirecting.

Reply:
@ericrallen : I have a tweak I'm going to implement on the Proxy itself :).
Once the PR is merged, I'll add the code.